maidr/widget/shiny.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from shiny.render import ui
+from shiny.types import Jsonifiable
+
+import maidr
+
+
+class RenderMaidr(ui):
```

Comment:
@krishnanand5 shiny breaks the conventional Pascal case for classes, since this will be used as a decorator, like `@render_maidr`. There have followe this convention for all their renderers. So, we should also follow their convention.

cc: @jooyoungseo

Reply:
Thank you for the confirmation @SaaiVenkat. I thought our repository's naming convention superseded supporting libraries but you are right. Better to follow the naming convention of the supporting libraries. I have reverted the class name to `snake_case`